conserver-src/conserver/main.c

```diff
@@ -1214,6 +1214,48 @@ VerifyEmptyDirectory(char *d)
 }
 #endif
 
+/*
+ * force a daily log rotation by raising SIGUSR2 when date changes
+ */
+static void*
+logRotator(void* arg)
+{
+    pthread_t main_thread = *(pthread_t*)arg;
+    time_t now = time(NULL);
+    struct tm now_tm;
+    int day, old_day;
+    struct timespec delay;
+    delay.tv_sec = 60;
```

Comment:
polling every 60 seconds to check if the day has changed seems a bit much - can we do every 10 mins or something or even every hour? Though saying that I doubt it's much of a performance hit anyway

Reply:
It's not a big cpu load, also if we make it ever 10 minutes or an hour then data will go into the wrong log file for a period, so i just made it a minute to minimise this